src/galileo/resources/api/datasets/bulk_delete_datasets_datasets_bulk_delete_delete.py

```diff
@@ -34,17 +37,14 @@ def _parse_response(
     *, client: ApiClient, response: httpx.Response
 ) -> Optional[Union[BulkDeleteDatasetsResponse, HTTPValidationError]]:
     if response.status_code == 200:
-        response_200 = BulkDeleteDatasetsResponse.from_dict(response.json())
```

Comment:
So it generates with `if response.status_code == :`
Then the script runs to convert to `if response.status_code == 200:`

Then the linter makes this change?